service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/UploadC2DocumentsSubmittedControllerTest.java

```diff
@@ -100,11 +103,69 @@ void submittedEventShouldNotifyCtscAdminWhenCtscToggleIsEnabled() throws Excepti
         );
     }
 
+    @Test
+    void submittedEventShouldNotifyAdminWhenC2IsNotUsingPbaPayment() throws Exception {
+        postSubmittedEvent(enableSendToCtscOnCaseDetails(NO, NO));
+
+        verify(notificationClient).sendEmail(
+            C2_UPLOAD_PBA_PAYMENT_NOT_TAKEN_TEMPLATE,
+            "admin@family-court.com",
+            expectedPbaPaymentNotTakenNotificationParams(),
+            CASE_ID.toString()
+        );
+
+        verify(notificationClient, never()).sendEmail(
+            C2_UPLOAD_PBA_PAYMENT_NOT_TAKEN_TEMPLATE,
+            "FamilyPublicLaw+ctsc@gmail.com",
+            expectedPbaPaymentNotTakenNotificationParams(),
+            CASE_ID.toString()
+        );
+    }
+
+    @Test
+    void submittedEventShouldNotifyCtscAdminWhenC2IsNotUsingPbaPaymentAndCtscToggleIsEnabled() throws Exception {
+        postSubmittedEvent(enableSendToCtscOnCaseDetails(YES, NO));
+
+        verify(notificationClient, never()).sendEmail(
+            C2_UPLOAD_PBA_PAYMENT_NOT_TAKEN_TEMPLATE,
+            "admin@family-court.com",
+            expectedPbaPaymentNotTakenNotificationParams(),
+            CASE_ID.toString()
+        );
+
+        verify(notificationClient).sendEmail(
+            C2_UPLOAD_PBA_PAYMENT_NOT_TAKEN_TEMPLATE,
+            "FamilyPublicLaw+ctsc@gmail.com",
+            expectedPbaPaymentNotTakenNotificationParams(),
+            CASE_ID.toString()
+        );
+    }
+
+    @Test
+    void submittedEventShouldNotNotifyAdminWhenUC2IsUsingPbaPayment() throws Exception {
+        postSubmittedEvent(enableSendToCtscOnCaseDetails(NO, YES));
+
+        verify(notificationClient, never()).sendEmail(
+            C2_UPLOAD_PBA_PAYMENT_NOT_TAKEN_TEMPLATE,
+            "admin@family-court.com",
+            expectedPbaPaymentNotTakenNotificationParams(),
+            CASE_ID.toString()
+        );
+
+        verify(notificationClient, never()).sendEmail(
+            C2_UPLOAD_PBA_PAYMENT_NOT_TAKEN_TEMPLATE,
+            "FamilyPublicLaw+ctsc@gmail.com",
+            expectedPbaPaymentNotTakenNotificationParams(),
+            CASE_ID.toString()
+        );
```

Comment:
I think we should go for more generic verification here. It'd make sense to leave `C2_UPLOAD_PBA_PAYMENT_NOT_TAKEN_TEMPLATE`, but replace all other arguments with `any()`. Also we could `verify` only once that way.